ServerTCPWpfApp/MainWindow.xaml.cs

```diff
@@ -12,13 +13,13 @@ namespace ServerTCPWpfApp;
 /// </summary>
 public partial class MainWindow : Window
 {
-    private const int Port = 8910;
-    private const int MaxConnections = 16;
-    private const int BufferSize = 256;
-    private static readonly IPHostEntry IpHost = Dns.GetHostEntry("localhost");
-    private readonly IPAddress _ipAddress = IpHost.AddressList[0];
-    private IPEndPoint? _iPEndPoint;
-    private Socket? _listenSocket;
+    private const int _port = 8910;
```

Comment:
Constants are always capitalized without underscores.
Rename to `Port`.

Reply:
Right, didn't notice it was a constant.